### ISSUE TITLE
MusicBrainz support

### ISSUE THREAD

**Issue by spl0k** [spl0k/supysonic#80]
See how we could integrate with MusicBrainz.
The first thing that comes to mind is using it to merge different artist names (artist credit) in one single main name.

**Comment by ivan-avalos** [comment]
Or maybe support scrobbling via ListenBrainz? It has a Last.fm-compatible API.

**Comment by spl0k** [comment]
Yeah I have a number of ideas on *Brainz integration, including this one. I'm just a bit lazy to bring them in 😅 (look at the date I created the issue)
